package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "SlickGrid",
-  "version": "3.5.0",
+  "version": "3.5.1",
+  "scripts": {
+    "build": "rollup -f iife -o slick.grid.js src/grid/index.js"
+  },
   "devDependencies": {
     "browser-sync": "^1.6.0",
     "coffee-script": "^1.8.0",
-    "gulp": "^3.8.9"
+    "gulp": "^3.8.9",
```

Comment:
OK (I never liked it anyway), I'll 'nix it and if anyone complains, it's easy to put back.